apps/condo/domains/notification/adapters/redStore/redStoreNotificationSender.js

```diff
@@ -0,0 +1,69 @@
+const { get } = require('lodash')
+
+
+const { fetch } = require('@open-condo/keystone/fetch')
+
+const { APS_RESPONSE_STATUS_SUCCESS } = require('../apple/constants')
+
+
+class RedStoreNotificationSender {
+    _projectId = null
+    _serviceToken = null
+    _url = null
+
+    /**
+     * @param config
+     */
+    constructor (config) {
+        this._projectId = config['project_id']
```

Comment:
What going to happening in case if project_id will be omit? Not sure, but probably better to add some guards for such cases

Reply:
added checks and `throw Error`

Reply:
it must be a first line of code - like a first step is validation of input parameters